doc/tools/docgram/orderedGrammar

```diff
@@ -1960,6 +1961,7 @@ or_and_intropattern_loc: [
 
 as_or_and_ipat: [
 | "as" or_and_intropattern_loc
+| "as" equality_intropattern
```

Comment:
This extra line is only for generating an error message at parsing time. I would say that it should not be documented (even if there is a discussion at #13205).

Reply:
Sure.  `as_or_and_ipat` is in the tactics chapter, which hasn't been updated yet.  We'll handle it then.